src/platforms/javascript/common/session-replay/custom-instrumentation/index.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Custom Instrumentation
```

Comment:
nit (feel free to ignore): Is "Custom Instrumentation" the correct title here? To me, the content here seems to be more around additional advanced configuration than doing custom things (maybe apart from the manual `Replay` start/stop section). No strong opinion here, though. We can also revisit this later.